tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -77,6 +77,37 @@ public VoteResolver(VoterInfoList vil) {
 
     private static final boolean DEBUG = false;
 
+    /**
+     * This enables a prose discussion of the voting process.
+     */
+    private StringBuilder transcript = null;
```

Comment:
suggestion: next step, move this to a new separate class and file -- maybe "VoteResolverTranscript.java"

Reply:
i meant "VoteResolverSplainer.java"

Reply:
seriously though, it should be a separate file for max encapsulation

Reply:
Actually I'd rather it was right here with the code that produces the calculation.

Reply:
it's trying to explain to humans what getWinningValue actually does.